deploy/roles/beaker/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+# main.yml - Beaker installation
+---
+- name: Ensure beaker directory is available
+  file:
+    path: "{{ beaker_src_dir }}"
+    state: directory
+    mode: 0755
+  tags:
+    - beaker
+
+- name: Download Beaker
+  get_url:
+    url: "{{ beaker_mirror }}"
+    dest: "{{ beaker_src_dir }}"
+  tags:
+    - beaker
+
+- name: Unarchive zipped beaker file
+  unarchive:
+    src: "{{ beaker_zip_file }}"
+    dest: "{{ beaker_parent_dir }}"
+    copy: no
+  tags: 
+    - beaker
+
+- name: Set up beaker environmental variables
+  template:
+    src: beaker-env.sh.j2
+    dest: "/etc/profile.d"
+    mode: 0644
+  tags:
+    - beaker
+
+
+
```

Comment:
In the docs (http://beakernotebook.com/getting-started), it mentions an "install-dependencies.sh" script that is designed for Ubuntu.  I doubt it will work out of the box on Centos, so we may have to re-implement that script in Ansible.  Could you take a look at the script and see what's in there, to determine if that is needed?  Also, we may want to create an entry in /usr/share/applications so that Beaker shows up in the Gnome Applications menu.